realm/realm-jni/src/util.cpp

```diff
@@ -185,21 +185,6 @@ jclass GetClass(JNIEnv* env, const char* classStr)
     return myClass;
 }
 
-void jprint(JNIEnv *env, char *txt)
```

Comment:
This has been useful for debugging in the past. But I assume we have another way now?